src/datasets/config.py

```diff
@@ -202,6 +202,7 @@
 LICENSE_FILENAME = "LICENSE"
 METRIC_INFO_FILENAME = "metric_info.json"
 DATASETDICT_JSON_FILENAME = "dataset_dict.json"
+METADATA_CONFIGS_FIELD = "configs_kwargs"
```

Comment:
e.g. `dataset_info` can be a dictionary to represent the `DatasetInfo` of the default config - but you can also define `dataset_info` as a list if you want to specify one `DatasetInfo` per config

Since most datasets only have one config, it was chose to be made `dataset_info` singular

Here `builder_config` defines the `BuilderConfig`, but you can also define a list if you want multiple configs